.github/workflows/changedPaths.nix

```
@@ -0,0 +1,58 @@
+updPath: basePath:
+
+let
+  releaseOpts = {
+    config = { };
+    allowUnfree = true;
+    preOverlays = [ ];
+    postOverlays = [ ];
+    buildVariants = false;
+    pin = true;
+  };
+
+  pkgs = import (builtins.fetchTarball "https://github.com/NixOS/nixpkgs/archive/refs/heads/nixos-22.11.tar.gz") { };
+
+  lib = pkgs.lib;
+
+  # Ignored outputs, to be removed additionally from the attribute set.
+  auxIgnore = [
+    "channel"
+    "nixexprs"
+  ];
+
+  # Recursive, flattened difference of attribute sets.
+  recAttrSetDiff = l: r: lib.attrsets.filterAttrsRecursive
+    (lK: lV: !(builtins.elem lK auxIgnore) && (if lib.hasAttr lK r
+    then lV != r."${lK}"
+    else true)
+    )
+    l
+  ;
+
+  # Simplify package set
+  simplify = as: with lib.attrsets; mapAttrs (_: v: v.drvPath) (filterAttrs (_: v: isDerivation v) as);
```

Statix suggestion ("Found eta-reduction"):
```suggestion
  simplify = as: with lib.attrsets; mapAttrs (_: v: v.drvPath) (filterAttrs (_: isDerivation) as);
```

---

Ah my old enemy, the eta reduction ^^ Hlint for haskell does the same to me and I tend to disagree. I don't like point free functions. I would prefer to keep this as is.